builtin/providers/azurerm/provider_test.go

```diff
@@ -0,0 +1,58 @@
+package azurerm
+
+import (
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+var testAccProviders map[string]terraform.ResourceProvider
+var testAccProvider *schema.Provider
+
+const (
+	testAccSecurityGroupName = "terraform-security-group"
+	testAccHostedServiceName = "terraform-testing-service"
+)
+
+// testAccStorageServiceName is used as the name for the Storage Service
+// created in all storage-related tests.
+// It is much more convenient to provide a Storage Service which
+// has been created beforehand as the creation of one takes a lot
+// and would greatly impede the multitude of tests which rely on one.
+// NOTE: the storage container should be located in `West US`.
+var testAccStorageServiceName = os.Getenv("AZURE_STORAGE")
```

Comment:
Currently we don't use this and it can be deleted until such time as we have an ARM storage set of resources. I'd argue that the tests should probably set up any required containers on their own anyway - it seems overly onerous and out of character with other acceptance tests to require upfront configuration beyond credentials.